code/game/objects/items/devices/defib.dm

```diff
@@ -4,6 +4,7 @@
 	desc = "This KHI-branded defib kit is a semi-automated model. Remove pads, slap on chest, wait."
 	icon = 'icons/obj/device.dmi'
 	icon_state = "defib_kit"
+	item_state = "defib_kit"
```

Comment:
In this case it was because the defib icon_state is changed to several very slightly different 'open' versions, but I've since realised none of those versions can actually be picked up, I'll take this line out.